lib/ex_aws/instance_meta.ex

```diff
@@ -45,6 +49,11 @@ defmodule ExAws.InstanceMeta do
     end
   end
 
+  def get_request_headers(config) do
+    # TODO: Make this conditional on something in config
```

Comment:
I'm working on this.  If the host isn't configured to enable IMDSv2 (and thus the metadata token API), this will fail.  I'm assuming we want to add a config option in ExAws to control whether we want to attempt to get a token.

Reply:
That seems reasonable.